react-ui/src/functions/stringToNumberArray.ts

```diff
@@ -0,0 +1,7 @@
+export const stringToNumberArray = (value: any): number[] => {
```

Comment:
any type is not allowed and the linter is giving warnings for these

Reply:
helper removed and case handled differently.